app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'benefits_intake_service/service'
+
+class BenefitsIntakeStatusJob
+  include Sidekiq::Job
+
+  def perform
+    pending_form_submission_ids = FormSubmission
+                                  .joins(:form_submission_attempts)
+                                  .where('form_submission_attempts.aasm_state = ?', 'pending')
+                                  .map(&:benefits_intake_uuid)
+    response = BenefitsIntakeService::Service.new.get_bulk_status_of_uploads(pending_form_submission_ids)
+    handle_response(response)
+  end
+
+  private
+
+  def handle_response(response)
+    response.body['data'].each do |submission|
+      if submission.dig('attributes', 'status') == 'error' || submission.dig('attributes', 'status') == 'expired'
```

Comment:
Both `error` and `expired` are considered error states. `expired` will be interesting to work with because I'm not sure if we'll be able to remediate them since we may not have form data?